### PR TITLE
Smoke test & sample content update

### PR DESCRIPTION
**Pull request by nate-double-u**
Using [Kubernetes Smoke Test page](https://github.com/kubernetes/website/blob/master/content/en/docs/test.md) as a start, adding a HNS test page. Link to follow when the deploy preview builds.

Reorganizing the Sample content as an example of how the /docs/ file system works (especially re: _index.md files)

_edit_ Deploy preview docs smoke test page: https://deploy-preview-57--cncf-hugo-starter.netlify.app/docs/test

Fixes: https://github.com/cncf/hugo-netlify-starter/issues/54

### CODE REVIEW

**Comment by nate-double-u** [comment]
@celestehorgan, could you please look at how I'm doing attributions, I'd like to make sure I'm doing them correctly.

**Comment by nate-double-u** [comment]
Docs smoke test page deploy preview:
https://deploy-preview-57--cncf-hugo-starter.netlify.app/docs/test

File: assets/sass/custom.sass [hunk 1]
```diff
@@ -98,3 +98,32 @@ nav
     padding-left: 0
     ul
       padding-left: 1.5rem
+
+// blockquotes and callouts
+// based on https://github.com/kubernetes/website/blob/18634747c956944d8a506a23bb00aafef68ea9ae/assets/scss/_custom.scss#L298-L328
+// converted from scss to sass via https://scss2sass.herokuapp.com/converter
```
Comment (celestehorgan) [inline assets/sass/custom.sass:104]:
```suggestion
```

File: content/docs/_index.md [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
+Fugiat sit enim reprehenderit occaecat ex dolore commodo irure ea in pariatur. Sint dolore proident non proident incididunt enim. Commodo ex deserunt ipsum laboris minim officia labore. Reprehenderit elit dolore ipsum sunt ea id amet. Ea laboris sint reprehenderit ullamco ullamco Lorem amet. Cillum commodo proident amet dolore labore veniam velit eiusmod velit dolore. Adipisicing veniam ullamco aute nulla enim aliqua officia sint do ullamco incididunt nisi ad fugiat.
 
-
-
-
-
-
-
-
+Generated text created using [knicklabs/lorem-ipsum.js](https://github.com/knicklabs/lorem-ipsum.js).
```
Comment (celestehorgan) [inline content/docs/_index.md:13]:
```suggestion
```

File: content/docs/docs-section-1/_index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+#### Subsubsection 1 with a very very long title
+
+Hello world `code`
+
+```shell
+codeblock
+```
+
+Generated text created using [knicklabs/lorem-ipsum.js](https://github.com/knicklabs/lorem-ipsum.js).
```
Comment (celestehorgan) [inline content/docs/docs-section-1/_index.md:41]:
```suggestion
```

File: content/docs/test.md [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+{{< /caution >}}
+
+
+{{< warning >}}
+Warnings point out something that could cause harm if ignored.
+{{< /warning >}}
+
+## Katacoda Embedded Live Environment
+
+{{< kat-button >}}
```
Comment (celestehorgan) [inline content/docs/test.md:328]:
```suggestion
```

**Review by celestehorgan — COMMENTED** [review]
Aside from the removals listed (remove the "generated by.." on all pages, fyi) this is lgtm!

File: content/docs/test.md (hunk 4 above)
Comment (nate-double-u) [reply content/docs/test.md:328]: No to `Katacoda`? I was super excited to see it, it's really nifty tool... I _suppose_ we don't need it here...

**Pull request merged**